CMakeLists.txt

```diff
@@ -255,7 +255,7 @@ option(OPENSCAP_ENABLE_SHA1 "Enable using the SHA-1 algorithm" ON)
 option(OPENSCAP_ENABLE_MD5 "Enable using the MD5 algorithm" ON)
 
 # INDEPENDENT PROBES
-cmake_dependent_option(OPENSCAP_PROBE_INDEPENDENT_ENVIRONMENTVARIABLE "Independent environmentvariable probe" ON "ENABLE_PROBES_INDEPENDENT; NOT WIN32" OFF)
```

Comment:
Amazing!

Please move it away from the list of `Independent probes incompatible with WIN32`.

Reply:
@jan-cerny  Thank you for reviewing the code. Fixed in commit  : https://github.com/OSCAPC/openscap/tree/01143823c9870a8b5b11dd427339ca0ce1c9a04a .